modules/nickserv/verify.c

```diff
@@ -235,6 +238,15 @@ static void ns_cmd_fverify(sourceinfo_t *si, int parc, char *parv[])
 	}
 }
 
+static void on_user_identify(user_t *u)
+{
+	myuser_t *mu = u->myuser;
+
+	if (MU_WAITAUTH & mu->flags)
+		notice(nicksvs.nick, u->nick, _("You have \2NOT COMPLETED\2 registration verification."));
+		//command_success_nodata(si, _("You have \2NOT COMPLETED\2 registration verification."), entity(mu)->name);
```

Comment:
This comment is unnecessary, especially when you consider the way the if-statement is written together with the indentation, someone uncommenting this will trigger this hook every time.